gedcom/people.py

```diff
@@ -147,6 +148,22 @@ def print_single(self):
         print("Single Individuals")
         print(table)
 
+    def print_recent_births(self):
+        """" US35 Print births in the last 30 days as list
+        """
+        people = self.individuals
+        table = PrettyTable(["ID", "Name", "Birthdate"])
+
+        for person_id in people:
+            individual = self.individuals[person_id]
+            if individual.get_birth_date is not None:
+                recent_date = self._current_time - timedelta(days=30)
+                if recent_date < individual.get_birth_date() and individual.get_birth_date() < self._current_time:
```

Comment:
recent_date and individual.get_birth_date() aren't the same types. same with individual.get_birth_date() and self._current_time.
There's an error when running the acceptance gedcom file.